requirements.txt

```diff
@@ -1,5 +1,5 @@
 jsonschema
-pygal
+plotly==4.8.1
```

Comment:
Remove specific version. So, that we always get latest version.

Reply:
I want to mark the version specifically until we add tests for HTML. Because it might be the case that new version implements some features differently.

Reply:
Most of the modules don't do backward incompatible changes so I guess it won't be a problem.